x/fswap/keeper/keeper_test.go

```diff
@@ -165,7 +169,8 @@ func (s *KeeperTestSuite) TestSwap() {
 			s.Require().NoError(err)
 
 			actualAmount := s.keeper.GetBalance(ctx, tc.from, s.swap.GetToDenom()).Amount
-			expectedAmount := tc.expectedBalanceWithoutMultiply.Mul(s.swap.SwapMultiple)
+			multipliedAmountDec := s.swap.SwapRate.Mul(sdk.NewDecFromBigInt(tc.expectedBalanceWithoutMultiply.BigInt()))
```

Comment:
The most critical logic in the swap would be the multiplication. It would be nice to provide the expected value in the cases as a literal.

Reply:
I extract calculation logic and fix a bug.
Please review again.